CodeEdit/AppDelegate.swift

```diff
@@ -25,11 +25,14 @@ final class CodeEditApplication: NSApplication {
 }
 
 @NSApplicationMain
-final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject { // swiftlint:disable:this type_body_length
+final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
     var updater: SoftwareUpdater = SoftwareUpdater()
 
     func applicationWillFinishLaunching(_ notification: Notification) {
         _ = CodeEditDocumentController.shared
+
+        // This enables window restoring on normal quit (instead of only on force-quit).
+        UserDefaults.standard.setValue(true, forKey: "NSQuitAlwaysKeepsWindows")
```

Comment:
This is more of an open suggestion, but we could wrap lines like these in a private function, describing what it does, instead of adding a comment above it. In this case it could be `enableWindowSizeSaveOnQuit()` for example. And we can always add a function description block for a better explanation.

Reply:
Good idea! I'll modify it